libmscore/measure.cpp

```diff
@@ -1351,6 +1322,9 @@ bool Measure::acceptDrop(EditData& data) const
                         viewer->setDropRectangle(canvasBoundingRect());
                   return true;
 
+            case ElementType::MEASURE_NUMBER:
+                  viewer->setDropRectangle(canvasBoundingRect());
+                  return true;
             case ElementType::BRACKET:
```

Comment:
adding an empty line here would match the sty of the other cases here, wouldn't it?